test/e2e/__snapshots__/ssr.test.ts.snap

```diff
@@ -242,6 +242,32 @@ exports[`browser (ssr: true) > none should render images 2`] = `
 ]
 `;
 
+exports[`browser (ssr: true) > prepr should render images 1`] = `
+[
+  "https://nuxt-prepr-demo.stream.prepr.io/q_5//4rhkcc7xzk7d-claymemoirscom-spider-pic.jpg",
```

Comment:
should this have two forward slashes in the middle? maybe this should be normalised?

Reply:
I will test your suggestion see if it addresses this issue. :+1: